MoltenVK/MoltenVK/GPUObjects/MVKBuffer.mm

```diff
@@ -212,6 +212,13 @@
     return _mtlBufferCache;
 }
 
+uint64_t MVKBuffer::getMTLBufferGPUAddress() {
+#if MVK_XCODE_14
+	return [_mtlBuffer gpuAddress] + getMTLBufferOffset();
```

Comment:
```suggestion
	return [getMTLBuffer() gpuAddress] + getMTLBufferOffset();
```

In my testing, I encountered situations where `_mtlBuffer` was not created at the time `getMTLBufferGPUAddress()` was called. The result was to return `0` from this call. I assume this happens in an app that accesses this function before the `VkBuffer` is populated, or submitted in a descriptor set. Lazily retrieving it using `getMTLBuffer()` fixes this.